### PR TITLE
remove the last ";" if the SQL is end with ";"

### PR DESCRIPTION
**Pull request by Flyfoxs**

### CODE REVIEW

File: libs/experimental/langchain_experimental/sql/base.py [hunk 1]
```diff
@@ -119,6 +119,8 @@ def _call(
                 callbacks=_run_manager.get_child(),
                 **llm_inputs,
             ).strip()
+            import re
```
Comment (eyurtsev) [inline libs/experimental/langchain_experimental/sql/base.py:122]: `re` can be pulled to global namespace

Would you be willing to create a helper function to clean up sql_cmd and a unit test for the clean up logic? it's common for the clean up logic to expand over time

Comment (Flyfoxs) [reply libs/experimental/langchain_experimental/sql/base.py:122]: where is the place to put the unit test, it has the depended on the connection to DB, could you give an example we already have.

Comment (Flyfoxs) [reply libs/experimental/langchain_experimental/sql/base.py]: Thanks, I already update the code as your suggestion.

**Comment by Flyfoxs** [comment]
Here is another blog report this issue

https://community.intersystems.com/post/langchain-fixed-sql-me

**Comment by leo-gan** [comment]
@Flyfoxs Hi , could you, please, resolve the merging issues? After that ping me and I push this PR for the review. Thanks!

**Comment by efriis** [comment]
Looks like this isn't being worked on anymore. I'll close for now, and if you resume work, feel free to reopen!

**Pull request closed without merging**